jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskdetails/TaskDetailsPresenter.java

```diff
@@ -131,6 +142,16 @@ public void onTaskSelectionEvent(@Observes final TaskSelectionEvent event) {
         this.currentTaskId = event.getTaskId();
         this.currentServerTemplateId = event.getServerTemplateId();
         this.currentContainerId = event.getContainerId();
+        this.currentTaskSummary =
```

Comment:
`refreshTask` calls the backend, so I'm not sure if just doing `setTaskDetails` would be enough then.